basic-sql-statements.md

```diff
@@ -122,18 +116,18 @@ To see all tables in the active database, use the `SHOW TABLES` statement:
 SHOW TABLES;
 ~~~
 ~~~
-+--------+
-| Table  |
-+--------+
-| table1 |
-| table2 |
-+--------+
++----------+
+|  Table   |
++----------+
+| accounts |
+| users    |
++----------+
 ~~~
 
 To view tables in a database that's not active, use `SHOW TABLES FROM` followed by the name of the database:
 
 ~~~ sql
-SHOW TABLES FROM db2;
+SHOW TABLES FROM animals;
```

Comment:
Oh I thought it was a reference to a database created in this document. No problem with the silly name, just thought it was a typo.